packages/core/src/modules/credentials/formats/jsonld/JsonLdCredentialFormat.ts

```diff
@@ -11,15 +11,29 @@ export interface SignCredentialOptionsRFC0593 {
   options: JsonLdOptionsRFC0593
 }
 
+export interface SignCredentialOptionsRFC0593AsJson {
+  credentialAsJson: JSON
+  options: JsonLdOptionsRFC0593
+}
+
+// use empty object in the acceptXXX jsonld format interface so we indicate that
+// the jsonld format service needs to be invoked
+type EmptyObject = Record<string, never>
+
+// it is an option to provide the verification method in acceptRequest
+type RequestVerificationMethod = {
```

Comment:
Can we call this `JsonLdCreateRequestFormat`? This makes it consistent with the indy credential format naming

Also, can we export it and make it an interface for consistency